main.py

```diff
@@ -1,10 +1,61 @@
+import json
+
 import init_django_orm  # noqa: F401
 
 from db.models import Race, Skill, Player, Guild
 
 
 def main() -> None:
-    pass
+    with open("players.json") as file:
+        player_data = json.load(file)
+        for player, player_data in player_data.items():
+            race_data = player_data["race"]
+            guild_data = player_data["guild"]
+
+            if Race.objects.filter(
+                    name=race_data["name"]
+            ).exists():
+                race = Race.objects.get(
+                    name=race_data["name"]
+                )
+            else:
+                race = Race.objects.create(
+                    name=race_data["name"],
+                    description=race_data["description"]
+                )
```

Comment:
You can use https://docs.djangoproject.com/en/4.1/ref/models/querysets/#get-or-create
It helps to reduce number of queries

Reply:
The same in other places